js/plugins.js

```diff
@@ -280,6 +280,25 @@ plugins.factory('userPlugins', function() {
     imagePlugin.name = 'image';
 
     /*
+     * mp4 video Preview
+     */
+    var videoPlugin = new Plugin(function(message) {
+
+        var url = message.match(urlRegexp);
+        var content = null;
+
+        if (url) {
+            url = url[0]; /* Actually parse one url per message */
+            if (url.match(/\.(mp4|webm|ogv)$/)) {
+                return '<video width="100%"><source src="'+url+'"></source></video>';
```

Comment:
100%? Doesn't that make the video huuuuge? I think I'd prefer a Reddit Enhancement Suite-like small default size with click&drag to resize. In any case 100% seems like a bad default to me